core/src/testFixtures/java/org/jobrunr/JobRunrAssertions.java

```diff
@@ -87,8 +86,10 @@ public static DatabaseAssertions assertThat(DataSource dataSource) {
     }
 
     public static String contentOfResource(String resourceName) {
-        try {
-            return Files.readString(Paths.get(JobRunrAssertions.class.getResource(resourceName).toURI()));
+        try (InputStream inputStream = JobRunrAssertions.class.getResourceAsStream(resourceName)) {
```

Comment:
the old approach failed after moving the resources to the `testFixtures` source set